Sloth.Web/Controllers/JobsController.cs

```diff
@@ -285,5 +242,19 @@ public async Task<IActionResult> RunCybersourceBankReconcile(DateTime date)
 
             return RedirectToAction(nameof(CybersourceBankReconcileDetails), new { id = record.Id });
         }
+
+        private static void SanitizeCybersourceBankReconcileJobsFilter(CybersourceBankReconcileJobsFilterModel model)
+        {
+            var date = (model.Date ?? DateTime.Now).Date;
+
+            model.Date = new DateTime(date.Year, date.Month, 1);
+        }
+
+        private static void SanitizeKfsScrubberJobsFilter(KfsScrubberJobsFilterModel model)
```

Comment:
Yeah, I wasn't sure. Originally it was because the filter models were going to be different. But they're pretty much identical. Oops, forgot to remove search fields from the filter models. I'll go ahead and consolidate them...